openedx_certificates/generators.py

```diff
@@ -0,0 +1,153 @@
+"""
+This module provides functions to generate certificates.
+
+The functions prefixed with `generate_` are automatically detected by the admin page and are used to generate the
+certificates for the users.
+
+We will move this module to an external repository (a plugin).
```

Comment:
Making it a plugin and putting it in a separate python package is fine, but moving it to a separate repo too early will only slow down development imho.